CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This file is used to list changes made in each version of the ntp cookbook.
 
 ## Unreleased
 
+## 3.11.1 - *2021-11-08*
```

Comment:
Hey,

Can you leave this out and just put things under `## Unreleased` the automated system will take care of the rest

Reply:
Hi I will do so, sorry never about that.

Reply:
I have submitted the change, please let me know if that is correct.

Reply:
Yep, that's it!